docs/setup_eecs281.md

```diff
@@ -82,6 +80,53 @@ If you've used Visual Studio on your computer before, start the tutorial at the
 
 If Visual Studio is new to you or new to your computer, start at the [beginning](https://eecs280staff.github.io/p1-stats/setup_visualstudio.html).  Stop after you've completed the [Add existing files](https://eecs280staff.github.io/p1-stats/setup_visualstudio.html#add-existing-files) section.
 
+#### `getopt` library
+Visual Studio does not ship with a `getopt` library, which processes command line arguments.  We'll install a copy.
+
+First, download [`getopt.c`](getopt.c) and [`getopt.h`](getopt.h).
+
+##### Option 1 (Recommended)
+Copy `getopt.c` into your project source code directory (e.g., `/Users/awdeorio/src/eecs281/p0-hello/src`).  You'll need to do this for every project.
+
+Copy `getopt.h` into one of the standard Visual Studio include folders. For Visual Studio 2022 Community, this folder is usually located in:
+```
+C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\VS\include
+```
+
+If you’re using a different version of VS (such as Community 2019), the location might be slightly different:
+```
+C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\VS\include
+```
+
+Open your file explorer and find the appropriate location shown above by starting at “This PC” (in the bottom left), and paste the file. This location is usually locked and can only be accessed if you are Administrator.
+
+Include the library in your program using brackets `<>`.
+```c++
+#include <getopt.h>
+```
+{: data-title="main.cpp" }
+
+##### Option 2
+Copy `getopt.c` and `getopt.h` into your project source code directory (e.g., `/Users/awdeorio/src/eecs281/p0-hello/src`).  You'll need to do this for every project.
```

Comment:
I agree that option 2 is much simpler than option 1, but it does have the ugly aspect having to change the `#include` each time, and it makes it easier for students to "loose" a submit.

I think that providing an `#ifdef` statement could be a good idea.

Reply:
Something about this second way just feels wrong, I feel like its better to do it the "right" way rather than a workaround that also has its own issues.

Reply:
Untested:
```c++
#ifdef _WIN32
#include "wingetopt.h"
#else
#include <getopt.h>
```

Reply:
I can test it locally on my machine. Would this mean that the `getopt.h` file should be renamed to `wingetopt.h`?

Reply:
I think so.  Try it both ways.  I'm getting a nasty problem on my machine (Apple/Clang) when I have a local .h file with the same as a standard library.

Reply:
I was able to get it to work locally on WSL and Visual Studio using both the `getopt.h` and `wingetopt.h` flavors.

I should be able to try it on the AG soon.

Reply:
The two file version with both a `.h` and a `.c` file worked on the AG.